shared/common-adapters/dropdown.tsx

```diff
@@ -37,43 +37,46 @@ type DropdownButtonProps = {
   inline?: boolean
   loading?: boolean
 }
-export const DropdownButton = (props: DropdownButtonProps) => (
-  <Kb.ClickableBox
-    onClick={!props.disabled ? props.toggleOpen : undefined}
-    style={Styles.collapseStyles([styles.dropdownBoxContainer, props.style])}
-  >
-    <Kb.Box2Measure
-      direction="vertical"
-      ref={props.popupAnchor}
-      className={Styles.classNames('dropdown_border', {
-        hover: !props.disabled,
-      })}
-      style={Styles.collapseStyles([
-        styles.measureBox,
-        {
-          paddingRight: props.inline
-            ? Styles.globalMargins.tiny
-            : Styles.isMobile
-              ? Styles.globalMargins.large
-              : Styles.globalMargins.small,
-        },
-        props.disabled ? {opacity: 0.3} : {},
-        {cursor: !props.disabled ? 'pointer' : undefined},
-        {width: props.inline ? undefined : '100%'},
-      ])}
+export const DropdownButton = (props: DropdownButtonProps) => {
```

Comment:
if a prop has a ref inside and you use props.blahRef then it gets confused about all prop values for some reason. destructure fixes this and is likely better anyways